code/development-env/docker/README.md

```diff
@@ -10,6 +29,8 @@ export USER_MANAGEMENT_API_CLIENT_ID= # from Auth0
 export USER_MANAGEMENT_API_CLIENT_SECRET= # from Auth0
```

Comment:
This is no longer needed, instead could you describe installing `direnv` and creating the file by taking it from Shared LastPass